listenbrainz/webserver/static/js/jsx/spotify-player.jsx

```diff
@@ -63,7 +64,10 @@ export class SpotifyPlayer extends React.Component {
       },
     })
     .then(response =>{
-      if(response.status === 403 || response.status === 401){
+      if(response.status === 401){
```

Comment:
You were worried about this working well yesterday. I trust you're happy with it now?

Reply:
After more testing and docs reading, yes, I think it should work.